Bottom: 09a1f6109a8c508e866ebeebc64340409dc05d66
Top:    525658b8e54c99e42aa11437f50c60d984069084
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-01-06 23:12:39 +0000

lesstif: Use Coord type for line-width, not int


---

diff --git a/src/hid/lesstif/main.c b/src/hid/lesstif/main.c
index c4c31ee..125a81b 100644
--- a/src/hid/lesstif/main.c
+++ b/src/hid/lesstif/main.c
@@ -52,7 +52,7 @@ typedef struct lesstif_gc_struct
 
   Pixel color;
   const char *colorname;
-  int width;
+  Coord width;
   EndCapStyle cap;
   char xor_set;
   char erase;
